mirage/scenarios/default.ts

```diff
@@ -157,7 +157,11 @@ function dashboardScenario(server: Server, currentUser: ModelInstance<User>) {
     for (const node of nodes.slice(4, 10)) {
         server.create('contributor', { node, users: currentUser, index: 11 });
     }
+
+    // NOTE: Some institutions are already created by this point
     server.createList('institution', 20);
+    // Create a specific institution to test institutional dashboard with; should be ID 29 at this point
+    server.create('institution', 'withInstitutionalUsers');
```

Comment:
please give this a specific id, e.g.
```suggestion
    server.create('institution', { id: 'demo'}, 'withInstitutionalUsers');
```